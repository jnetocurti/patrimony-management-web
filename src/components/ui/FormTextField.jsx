import React, { memo } from 'react';

import { TextField } from '@material-ui/core';

const FormTextField = memo((props) => {
  return (
    <TextField {...props} required fullWidth size="small" variant="outlined" />
  );
});

export default FormTextField;
