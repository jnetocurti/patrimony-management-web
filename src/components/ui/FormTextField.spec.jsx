import React from 'react';

import { shallow } from 'enzyme';

import FormTextField from './FormTextField';

describe('src/components/ui/TextFields', () => {
  it('should render correctly', () => {
    const component = shallow(<FormTextField />);
    expect(component).toMatchSnapshot();
  });
});
