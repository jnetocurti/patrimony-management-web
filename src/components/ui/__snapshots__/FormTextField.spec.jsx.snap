// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/ui/TextFields should render correctly 1`] = `
<WithStyles(ForwardRef(TextField))
  fullWidth={true}
  required={true}
  size="small"
  variant="outlined"
/>
`;
